airflow/contrib/hooks/gcp_api_base_hook.py

```diff
@@ -161,6 +214,25 @@ def _get_field(self, f, default=None):
     def project_id(self):
         return self._get_field('project')
 
+    @staticmethod
+    def quota_retry(*args, **kwargs):
+        """
+        Function decorator that provides a mechanism to repeat requests in response to exceeding a temporary
+        quote limit.
+        """
+        def decorator(fun):
```

Comment:
Adding `functools.wraps` will preserve function name and docs.
```suggestion
       @wraps
        def decorator(fun):
```

Reply:
It's only works decorator not decorator factory.